### ISSUE TITLE
[Feature] Keyboard shortcuts for skipping intros and jumping between episode

### ISSUE THREAD

**Issue by NadCat1** [Moonfin-Client/Moonfin-Core#1335]
### What problem does this solve?

I love how much Moonfin is already usable with a keyboard only, it's almost a perfect replacement for Plex HTPC. The only two things I'm missing for a 100% mouse-free experience are being able to skip intros and jump between episodes.

### Proposed solution

Going by [the wiki](https://github.com/Moonfin-Client/Moonfin-Core/wiki/User-Guide#keyboard-shortcuts-desktop), the s button doesn't have any function and could be used for skipping intros. Personally, I would prefer being able to use enter (meaning no skip intro button shown -> enter opens the overlay; skip intro button shown -> enter skips intro). As for jumping between episodes, Plex HTPC allows you to select the previous/next episode buttons with the arrow key as long as the overlay is visible (without the overlay, the arrow keys are used for seeking). That seems like the most intuitive approach to me. 

### Platform

Windows
